src/errors/runtime/type-assertions.js

```diff
@@ -15,8 +15,9 @@ function isNonNegativeValue (value) {
 
 export var is = {
     number: {
-        name:      'number',
-        predicate: isFiniteNumber
+        name:              'number',
+        predicate:         isFiniteNumber,
+        getActualValueMsg: (value, type) => type === 'number' ? value : type
```

Comment:
Please, make it clear that we filter `NaN` here:
```
getActualValueMsg: (value, type) => isNaN(value) ? value : type
```

Reply:
`INFINITY` is also possible